react/features/base/tracks/functions.js

```diff
@@ -216,6 +216,25 @@ export function isRemoteTrackMuted(tracks, mediaType, participantId) {
     return !track || track.muted;
 }
 
+/**
+ * Returns whether or not the current environment needs a user interaction with
+ * the page before any unmute can occur.
+ *
+ * @param {Object} state - The redux state.
+ * @returns {boolean}
+ */
+export function isUserInteractionRequiredForUnmute(state) {
+    const { browser } = JitsiMeetJS.util;
+
+    return !browser.isReactNative()
+        && !browser.isChrome()
+        && !browser.isChromiumBased()
+        && !browser.isElectron()
```

Comment:
Isn't it going yo be better if we put the browser checks in `BrowserCapabilities` object (`browser`) from lib-jitsi-meet? If one day we start using the capabilities.json there (which was the plan) this can fit very well there.

Reply:
Agreed.